packages/react-shortcuts/src/Shortcut/Shortcut.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import {contextTypes} from '../ShortcutProvider';
-import {Key} from '../types';
+import {Key} from '../keys';
```

Comment:
We should make `Key` the `default` export and keep the more specific one named exports I think.